### ISSUE TITLE
[Bug]: the way to vec the env by `make_vec_env(MyWrapper, n_envs=16)` comes error,but by `DummyVecEnv([MyWrapper] * 16)` goes well.

### ISSUE THREAD

**Issue by RookieMasterrr** [DLR-RM/stable-baselines3#1678]
### 🐛 Bug

when i vec the env by `make_vec_env(MyWrapper, n_envs=16`  then comes error behind.

but when i vec the env by `DummyVecEnv([MyWrapper] * 16)` everything goes well.

### To Reproduce

```python
import gymnasium as gym
from stable_baselines3 import DQN
from stable_baselines3.common.vec_env import DummyVecEnv
model = DQN(
        policy='MlpPolicy',
        # env=make_vec_env(MyWrapper, n_envs=16),
        env=(DummyVecEnv([MyWrapper] * 16)),
)
```

### Relevant log output / Error message

```shell
Traceback (most recent call last):
  File "C:\Users\RookieMaster\Desktop\different-algrm-ppo\main.py", line 40, in <module>
    env=make_vec_env(MyWrapper, n_envs=16),  
  File "D:\Software\Python\lib\site-packages\stable_baselines3\common\env_util.py", line 127, in make_vec_env
    vec_env.seed(seed)
  File "D:\Software\Python\lib\site-packages\stable_baselines3\common\vec_env\base_vec_env.py", line 281, in seed
    seed = np.random.randint(0, 2**32 - 1)
  File "mtrand.pyx", line 779, in numpy.random.mtrand.RandomState.randint
  File "_bounded_integers.pyx", line 1423, in numpy.random._bounded_integers._rand_int32
ValueError: high is out of bounds for int32

Process finished with exit code 1
```

### System Info

- OS: Windows-10-10.0.18363-SP0 10.0.18363
- Python: 3.9.13
- Stable-Baselines3: 2.1.0
- PyTorch: 2.0.1+cu117
- GPU Enabled: True
- Numpy: 1.25.2
- Cloudpickle: 2.2.1
- Gymnasium: 0.29.1
- OpenAI Gym: 0.26.2

### Checklist

- [X] My issue does not relate to a custom gym environment. (Use the custom gym env template instead)
- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have provided a [minimal and working](https://github.com/DLR-RM/stable-baselines3/issues/982#issuecomment-1197044014) example to reproduce the bug
- [X] I've used the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by qgallouedec** [comment]
What is `MyWrapper`?
It seems to me that this has been resolved in #1579. Can you upgrade sb3 to >=2.2 and see what happens?

**Comment by RookieMasterrr** [comment]
sorry for wasting you guys' time, i miss that issue
